tests/pkcli/ci1_test.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""test ci
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pkg_resources
+from pykern.pkdebug import pkdp
+import re
+import pytest
+
+
+def test_check_prints_dot_prefix():
+    from pykern.pkcli import ci, test
+    from pykern import pkunit
+    from pykern import pksetup
+
+    ci._EXCLUDE_FILES = re.compile(
```

Comment:
This should be an edited value, that is, start with the existing value and edit out the _WORK_DIR_SUFFIX VALUE. Useful fact: re.compile('my regex').pattern == 'my regex'.